build.gradle

```diff
@@ -37,7 +37,13 @@ configurations {
 }
 
 ext {
-  kafkaVersion = "3.8.1"
+  kafkaVersion = "3.9.1"
+
+  configurations.all {
+    resolutionStrategy {
+      force "org.apache.commons:commons-collections4:4.5.0"
```

Comment:
Add a comment here to explain why this is being done - i.e. what CVE is being addressed due to which earlier version of this library.